core/controllers/admin.py

```diff
@@ -80,63 +77,24 @@ def get(self):
         """Handles GET requests."""
         demo_exploration_ids = list(feconf.DEMO_EXPLORATIONS.keys())
 
-        recent_job_data = jobs.get_data_for_recent_jobs()
-        unfinished_job_data = jobs.get_data_for_unfinished_jobs()
         topic_summaries = topic_fetchers.get_all_topic_summaries()
         topic_summary_dicts = [
             summary.to_dict() for summary in topic_summaries]
-        for job in unfinished_job_data:
-            job['can_be_canceled'] = job['is_cancelable'] and any([
-                klass.__name__ == job['job_type']
-                for klass in (
-                    jobs_registry.ONE_OFF_JOB_MANAGERS + (
-                        jobs_registry.AUDIT_JOB_MANAGERS))])
-
-        queued_or_running_job_types = set([
-            job['job_type'] for job in unfinished_job_data])
-        one_off_job_status_summaries = [{
-            'job_type': klass.__name__,
-            'is_queued_or_running': (
-                klass.__name__ in queued_or_running_job_types)
-        } for klass in jobs_registry.ONE_OFF_JOB_MANAGERS]
-        audit_job_status_summaries = [{
-            'job_type': klass.__name__,
-            'is_queued_or_running': (
-                klass.__name__ in queued_or_running_job_types)
-        } for klass in jobs_registry.AUDIT_JOB_MANAGERS]
-
-        continuous_computations_data = jobs.get_continuous_computations_info(
-            jobs_registry.ALL_CONTINUOUS_COMPUTATION_MANAGERS)
-        for computation in continuous_computations_data:
-            if computation['last_started_msec']:
-                computation['human_readable_last_started'] = (
-                    utils.get_human_readable_time_string(
-                        computation['last_started_msec']))
-            if computation['last_stopped_msec']:
-                computation['human_readable_last_stopped'] = (
-                    utils.get_human_readable_time_string(
-                        computation['last_stopped_msec']))
-            if computation['last_finished_msec']:
-                computation['human_readable_last_finished'] = (
-                    utils.get_human_readable_time_string(
-                        computation['last_finished_msec']))
 
         feature_flag_dicts = feature_services.get_all_feature_flag_dicts()
 
+        config_properties = config_domain.Registry.get_config_property_schemas()
+
+        # Removes promo-bar related configs as promo-bar is handlded by
+        # release coordinators in /release-coordinator page.
+        del config_properties['promo_bar_enabled']
```

Comment:
Just to check -- I think, when we save config properties, we do so all at once. Is it a problem if some config properties have been deleted?

Reply:
Yeah, probably should fix this to do just 1 put_multi(). Maybe file an issue for the future?

Otherwise, this part LGTM, thanks for confirming.

Reply:
It's not straight forward as saving is happening after travelling 3 layers (controller/services/domain) and there's not a direct set_properties files for config.

I've created an issue (#12929) to do pulti multi and also only send changed values from frontend instead of all the values. (Is there any specific reason on why we set all value and not just the changed one?)

Reply:
Hmm, should we maybe add a TODO so that we can specifically include the info about who exactly can modify the config in the config model itself?

Reply:
I think the issue is that the "save" button is at the bottom so when it's clicked we don't know what's been changed and what hasn't been.

There may well be better ways to do this, I'm pretty open to anything as long as it works correctly.

Reply:
Sorry, what should be the TODO also where should I add this comment?